lib/selinuxtest.pm

```diff
@@ -121,7 +121,8 @@ sub set_sestatus {
 
     # enable SELinux in grub
     die "Need mode 'enforcing' or 'permissive'" unless $mode =~ /enforcing|permissive/;
-    add_grub_cmdline_settings('security=selinux selinux=1 enforcing=' . ($mode eq 'enforcing' ? 1 : 0), update_grub => 1);
+    replace_grub_cmdline_settings('lsm=apparmor', '', update_grub => 1);
+    add_grub_cmdline_settings('lsm=selinux security=selinux selinux=1 enforcing=' . ($mode eq 'enforcing' ? 1 : 0), update_grub => 1);
```

Comment:
Lili,
 kernel parameter can be recognized by both TW and SLES, right?

I am wondering that if the older releases. sles15sp3- can also support this para.
others lgtm

Reply:
Yes, lsm is valid for sles + TW. "lsm" is not a new thing sles15sp3 also support lsm.